Kernel/Language/pl.pm

```diff
@@ -2772,6 +2772,9 @@ sub Data {
         'Outgoing message' => 'Wiadomość wychodząca',
         'Internal message' => 'Wiadomość wewnętrzna',
         'Sending of this message has failed.' => '',
+        'Error while validating Recipient email address.' => 'Błąd w trakcie weryfikacji adresu odbiorcy.',
+        'Error while validating Message data.' => 'Błąd w trakcie weryfikacji danych wiadomości.',
+        'Error while validating Sender email address.' => 'Błąd w trakcie weryfikacji adresu nadawcy.',
```

Comment:
Hi @pboguslawski,

Translations are performed via [Weblate](https://translations.znuny.org/projects/znuny/znuny/
). 
Since these translations are not yet available, the corresponding entry in the .po file is missing here.

You can use this Dev::Tools::TranslationsUpdate script to generate the missing translations. 
But the script has not yet realized that it should include this in the translation...

I assume that the translations come from the file `Kernel/System/MailQueue.pm`.
Use the Translatable() function to mark a text as translatable in the Perl code.

```
use Kernel::Language qw(Translatable);

ErrorMessage => Translatable('Error while validating Message data.'),

ErrorMessage => Translatable('Error while validating Sender email address.'),

ErrorMessage => Translatable('Error while validating Recipient email address.'),
```

After that you can execute the script.

```
Make sure that all symbolic links are removed before you run this script.
bin/otrs.Console.pl Dev::Tools::TranslationsUpdate --generate-po
```

Reply:
I can see you've already added translations in

https://github.com/znuny/Znuny/commit/4304df3c491f43d9666983094e5c12066adb6069
https://translations.znuny.org/

so removed this part from this PR (https://github.com/znuny/Znuny/pull/131/commits/b0e899e9c68a05fb290410d90a250c67c05a2d9a).